src/js/main.js

```diff
@@ -181,14 +216,42 @@ increaseRoomsEl.addEventListener('click', function (){
   if(value3 < 30) {
     value3 += 1;
     updateRooms();
+    param.set('rooms', value3.toString());
+    url.search = param.toString();
+
   }
 });
 decreaseRoomsEl.addEventListener('click', function (){
   if(value3 > 1) {
     value3 -= 1;
     updateRooms();
+    param.set('rooms', value3.toString());
+    url.search = param.toString();
+
   }
 });
+buttonSearch.addEventListener('click', function() {
+  const fullUrl = getAll();
+  console.log(fullUrl);
+});
+
+function getAll() {
+  url.search = param.toString();
+  return url.toString();
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
 
```

Comment:
Too many line breaks. If you need to split logic, use different js files for that